appeals-service-api/src/controllers/appeals.js

```diff
@@ -5,6 +5,26 @@
 const AppealsSchema = require('../schemas/appeals');
 
 module.exports = {
+  async uploadFile(req, res) {
+    console.log(req)
```

Comment:
Remove `console.log` - there is a beautiful logger in `req.log` (or `./lib/logger` if don't have `req` available)